### DISCUSSION TITLE
My returned data is old

### DISCUSSION THREAD

**Discussion by Craftoncu** [bimmerconnected/bimmer_connected#503] [Q&A]
Good evening!

Does anybody have a clue why the MyBMW App always shows the correct, live data and the returned data by this lib is annotated with  "lastUpdatedAt": "2022-10-02T17:50:46Z"

I cant explain me this...

Thanks in advance!

Daniel

**Comment by rikroe** [comment]
Is it just the timestamp that is wrong or is the actual data old as well?

If you're seeing last timestamp, you probably use it from the CLI, right?
If so, do a fingerprint and check all the timestamps in the files (there are multiple).
